vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cp.go

```diff
@@ -244,6 +244,7 @@ func copyFromPod(f cmdutil.Factory, cmd *cobra.Command, cmderr io.Writer, src, d
 // stripPathShortcuts removes any leading or trailing "../" from a given path
 func stripPathShortcuts(p string) string {
 	newPath := path.Clean(p)
+	newPath = strings.TrimLeft(p, "../")
```

Comment:
pretty sure this is reverting https://github.com/openshift/origin/pull/20034, though I have questions about that pick

Reply:
That pick is in the origin-3.10-kubernetes-1.10.2 branch, but not picked into origin 3.10 vendor, so it either should be in this branch, or not in the origin-3.10-kubernetes-1.10.2 branch.

Reply:
Ah, it's picking the fix up from master.